lib/view/json/JsonLijstResponse.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+require_once 'view/JsonResponse.class.php';
+
+/**
+ * JsonLijstResponse.class.php
+ *
+ * @author G.J.W. Oolbekkink <g.j.w.oolbekkink@gmail.com>
+ *
+ */
+class JsonLijstResponse extends JsonResponse {
```

Comment:
Ok, neem dan mijn implementatie over zoals hieronder genoemd: https://github.com/csrdelft/csrdelft.nl/pull/41#r79328645

Deze is afgeleid van DataTableResponse die efficiënt met geheugen moet omgaan.